From fba847d3cebdbabc96aa7d6cbc3cc94db5994249 Mon Sep 17 00:00:00 2001
From: Jean Delvare <jdelvare@suse.de>
Date: Tue, 23 Sep 2025 13:42:10 +0200
Subject: [PATCH 40/45] dmidecode.8: Clarify what "bios" and "firmware"
 keywords refer to

When passing "bios" to a command line option, it means the platform
firmware, regardless of it technically implementing the BIOS standard.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
---
 man/dmidecode.8 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/man/dmidecode.8 b/man/dmidecode.8
index f25f9a0..c4c292d 100644
--- a/man/dmidecode.8
+++ b/man/dmidecode.8
@@ -108,6 +108,9 @@ It must be a keyword from the following list:
 .BR processor\-version ,
 .BR processor\-frequency .
 .hy
+In this context, for historical reasons, "bios" designates the platform
+firmware (regardless of it technically implementing the BIOS standard),
+while "firmware" designates the embedded controller firmware, if applicable.
 Each keyword corresponds to a given \s-1DMI\s0 type and a given offset
 within this entry type.
 Not all strings may be meaningful or even defined on all systems. Some
@@ -149,6 +152,8 @@ keyword from the following list:
 .BR slot .
 .hy
 Refer to the DMI TYPES section below for details.
+In this context, for historical reasons, "bios" designates the platform
+firmware, regardless of it technically implementing the BIOS standard.
 If this option is used more than once, the set of displayed entries will be
 the union of all the given types.
 If \fITYPE\fP is not provided or not valid, a list of all valid keywords
-- 
2.47.0

